scripts/env.sh

```diff
@@ -2,7 +2,7 @@
 export PORTIERIS_PULL_APIKEY=
 export PORTIERIS_TESTIMAGE_APIKEY=
 # charts to test
-export VERSION=v0.13.19
+export VERSION=
```

Comment:
Should there be a version here?

Reply:
I believe it should be blank and filled in along with the other blank fields. I removed it intentionally, after I missed it originally when filling out the blank fields in the file, and got errors for the wrong version number as it was hardcoded as that version. 
See `fill in the blanks` in [the runbook](https://pages.github.ibm.com/alchemy-registry/operational-docs/runbooks/cr/release_portieris/#action-to-take---test)